sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -470,6 +470,15 @@ def _to_rest_object(self) -> JobBase:
         # This will be resolved in job_operations _resolve_arm_id_or_upload_dependencies.
         component_id = self.component if isinstance(self.component, str) else self.component.id
 
+        # MFE not support pass None or empty input value. Remove the empty inputs and parameters in jobs.
+        empty_inputs = [k for k, v in built_inputs.items() if v is None or v == ""]
+        if empty_inputs:
+            built_inputs = {k: v for k, v in built_inputs.items() if k not in empty_inputs}
+            empty_inputs_value = [f"${{{{parent.inputs.{k}}}}}" for k in empty_inputs]
```

Comment:
add test case for empty pipeline job input

Reply:
Add test case for empty job input